src/main/java/com/google/api/codegen/java/JavaGapicContext.java

```diff
@@ -332,4 +332,34 @@ public String getMultilineHeading(String heading) {
   public JavaDocConfig.Builder newJavaDocConfigBuilder() {
     return JavaDocConfig.newBuilder();
   }
+
+  /**
+   * Get a Java formatted primitive value, given a primitive type and a string representation of
+   * that value. The value must be a valid example of that type. Values of type String and Bytes
+   * must contain quote characters. Values of type Bool must be either the string 'true' or 'false'
+   * (other capitalizations are permitted).
+   */
+  public String renderPrimitiveValue(TypeRef type, String value) {
+    Type primitiveType = type.getKind();
+    if (!PRIMITIVE_TYPE_MAP.containsKey(primitiveType)) {
+      throw new IllegalArgumentException(
+          "Initial values are only supported for primitive types, got type "
+              + type
+              + ", with value "
+              + value);
+    }
+    switch (primitiveType) {
```

Comment:
Oh! I didn't catch that. Maybe added a comment on the default case calling that out.